src/webui/webapplication.cpp

```diff
@@ -774,6 +777,33 @@ void WebApplication::action_command_bottomPrio()
     BitTorrent::Session::instance()->bottomTorrentsPriority(hashes);
 }
 
+void WebApplication::action_command_setLocation()
+{
+    CHECK_URI(0);
+    CHECK_PARAMETERS("hashes" << "location");
+
+    QStringList hashes = request().posts["hashes"].split("|");
```

Comment:
The legacy codebase (and coding guidelines) is all I have to go off of when deciding on a coding style. When a codebase is followed, it generally obviates code reviews like these. To an outsider trying to help out on the project, this review seems very much out of left field. This is why I try to follow the existing codebase.